api/handlers/metadata.go

```diff
@@ -60,45 +63,65 @@ type metadataHandler struct {
 func (h *metadataHandler) getMetadata() middleware.Responder {
 	logger.Info("API received a GET metadata event")
 
-	var namespace string
-	namespace = os.Getenv("POD_NAMESPACE")
+	namespace := os.Getenv("POD_NAMESPACE")
 
 	var payload models.Metadata
 	payload.Namespace = namespace
-	payload.Keptnversion = "N/A"
+	payload.Keptnversion = defaultVersion
 	payload.Keptnlabel = "keptn"
-	payload.Bridgeversion = "N/A"
+	payload.Bridgeversion = defaultVersion
 	payload.Shipyardversion = "0.2.0"
 
-	if h.k8sClient != nil {
-		deploymentsClient := h.k8sClient.AppsV1().Deployments(namespace)
-		bridgeDeployment, err := deploymentsClient.Get(context.TODO(), "bridge", metav1.GetOptions{})
-		if err != nil {
-			// log the error, but continue
-			logger.WithError(err).Error("Error getting deployment info")
-		} else {
-			payload.Bridgeversion = strings.TrimPrefix(bridgeDeployment.Spec.Template.Spec.Containers[0].Image, "keptn/")
-		}
+	if bridgeVersion, err := h.getBridgeVersion(namespace); err == nil {
+		payload.Bridgeversion = bridgeVersion
+	} else {
+		logger.WithError(err).Error("Error getting brdige version")
+	}
```

Comment:
```suggestion
	if bridgeVersion, err := h.getBridgeVersion(namespace); err != nil {
		logger.WithError(err).Error("Error getting brdige version")
	} else {
		payload.Bridgeversion = bridgeVersion
	}
```

Reply:
just a minor thing, usually we check for the existence of an error, and not for the existence of no error. It's just a bit weird to see it like this. :)